WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs

```diff
@@ -43,6 +43,7 @@ void OnClick()
 								return;
 							}
 
+							MainViewModel.Instance.NavBar.SelectedWallet = MainViewModel.Instance.NavBar.Wallets.FirstOrDefault(x => x.Wallet == wallet);
```

Comment:
Can we do:

```c#
// TODO: temp solution, remove model creation
var walletModel = new WalletModel(wallet);
UiContext.Navigate().To(walletModel);
```

So @ichthus1604 will see that we need to work with WalletModels here.

Reply:
I think @ichthus1604 can review this to see if it's the correct approach given the current status of the refactor. I've seen this workaround applied before, though :)

Reply:
I think @SuperJMN's is okay for a temporary workaround. WalletManagerViewModel needs to die on fire anyway.